server/util_test.go

```diff
@@ -399,3 +400,26 @@ func TestInterceptConfigsWithBadPermissions(t *testing.T) {
 		t.Fatalf("Failed to catch permissions error, got: [%T] %v", err, err)
 	}
 }
+
+func TestStartCmdwithEmptyandNonEmptyMinGasPrices(t *testing.T){
+
+	// first case - raise an error
+	tempDir := t.TempDir()
+	cmd := StartCmd(nil, tempDir)
+
+
+	// MinGasPrices ="0.22  -> no error raised
+	cfg := config.DefaultConfig()
```

Comment:
@amaurym  Just wonder how I can integrate this config with non empty MinGasPrices field into serverCtx?

Reply:
`cfg.BaseConfig.MinGasPrices ="0.22"` looks good.

We can have a simple table test:
- cfg.BaseConfig.MinGasPrices ="0.22" -> `err := cmd.ExecuteContext(ctx); require.NoError(err)`
- cfg.BaseConfig.MinGasPrices ="" -> `err := cmd.ExecuteContext(ctx); require.Error(err)`

Reply:
Ah, just re-read, I think you mean how to make the `start` cmd take the local `cfg` into account in this test. Let me think about it...

Reply:
> Ah, just re-read, I think you mean how to make the `start` cmd take the local `cfg` into account in this test. Let me think about it...

Yes, exactly!